semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -212,7 +212,7 @@ public static Optional<String> getFullHttpUrl(Event event) {
         : getValidHttpUrl(event).map(AttributeValue::getValue);
   }
 
-  public static Optional<String> getDestinationIpAddress(Event event) {
+  public static Optional<String> getPeerIpAddress(Event event) {
```

Comment:
What if there is a consumer of this method? I think, add a new public method `getPeerIpAddress` and reuse the existing method internally if required.

There was same comment above two.